presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -871,4 +872,17 @@ public FeaturesConfig setLegacyUnnestArrayRows(boolean legacyUnnestArrayRows)
         this.legacyUnnestArrayRows = legacyUnnestArrayRows;
         return this;
     }
+
+    public boolean isIgnoreStatsCalculatorFailures()
```

Comment:
Please put the methods in the same order as the properties (I'd expect this to be below `enableStatsCalculator`)

Reply:
All the methods has to be reordered in that class. I moved this very method after `enableStatsCalculator`, but methods before `enableStatsCalculator` are not in the correct order.